build/js/intlTelInput.js

```diff
@@ -631,11 +641,25 @@ https://github.com/Bluefieldscom/intl-tel-input.git
         },
         // decide where to position dropdown (depends on position within viewport, and scroll)
         _setDropdownPosition: function() {
-            var inputTop = this.telInput.offset().top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
+            var that = this, pos = this.telInput.offset(), inputTop = pos.top, windowTop = $(window).scrollTop(), // dropdownFitsBelow = (dropdownBottom < windowBottom)
             dropdownFitsBelow = inputTop + this.telInput.outerHeight() + this.dropdownHeight < windowTop + $(window).height(), dropdownFitsAbove = inputTop - this.dropdownHeight > windowTop;
             // dropdownHeight - 1 for border
             var cssTop = !dropdownFitsBelow && dropdownFitsAbove ? "-" + (this.dropdownHeight - 1) + "px" : "";
             this.countryList.css("top", cssTop);
+            // if container calculate postion and append this.$drop to container
+            if (this.options.container) {
+                var inputHeight = !dropdownFitsBelow && dropdownFitsAbove ? 0 : that.telInput.outerHeight();
+                // calculate placement
+                this.$drop.css({
+                    top: pos.top + inputHeight - parseInt(this.telInput.css("borderTopWidth"), 10) - parseInt(this.telInput.css("borderBottomWidth"), 10),
+                    left: pos.left
+                });
+                // close menu on resize/scroll
+                this.telInput.parents().off("resize" + this.ns + " scroll" + this.ns).on("resize" + this.ns + " scroll" + this.ns, function() {
```

Comment:
When the window is scrolled/resized, the menu will be in the wrong position unless the position is constantly recalculated on scroll/resize. This is doable, but more resource intensive and also can create some weird issues when the input is scrolled out of frame. The menu would still be visible even though the input isn't. Play around with scrolling the inputs with `container: 'body'`.

The underlying code in these examples is outdated, but the concepts are the same:

[When scrolling, menu position is not recalculated](http://embed.plnkr.co/lobCJwstCUnRHx5FxZ9s/preview)
[When scrolling, menu position is recalculated](http://embed.plnkr.co/vYqhcKJGKxWp6vUvLTHZ/preview)

I figured it would best to simply close the menu. I suppose this would be something we could implement as an option (`autoClose`?) for devs not using the `dropdownContainer` option, but I'm not sure how desirable it would be to use `autoClose` without also using `dropdownContainer`, and likewise, how desirable it would be to use `dropdownContainer` without `autoClose`.

I guess we could just make `autoClose` enabled by default if a `dropdownContainer` is set, unless the dev specifies that it should be disabled to allow for full flexibility.